src/operations/add_user.ts

```diff
@@ -18,11 +18,16 @@ export interface AddUserOptions extends CommandOperationOptions {
 }
 
 /** @internal */
-export class AddUserOperation extends CommandOperation<AddUserOptions, Document> {
+export class AddUserOperation extends CommandOperation implements AddUserOptions {
   db: Db;
   username: string;
   password?: string;
 
+  /** @deprecated Please use db.command('createUser', ...) instead for this option */
+  digestPassword?: null;
```

Comment:
Can you add the deprecated props to the bottom of this list? This can read as `digestPassword` is the only deprecated, or all that follow are as well.